inject.js

```diff
@@ -12,15 +12,14 @@ var MIN = 60*SEC
 module.exports = function (name, override) {
   name = name || 'ssb'
   var HOME = home() || 'browser' //most probably browser
-  var port = override && override.port || 8008
-  return RC(name || 'ssb', merge({
+  var result = RC(name || 'ssb', merge({
     //just use an ipv4 address by default.
     //there have been some reports of seemingly non-private
     //ipv6 addresses being returned and not working.
     //https://github.com/ssbc/scuttlebot/pull/102
     party: true,
     host: nonPrivate.v4 || '',
-    port: port,
+    port: 8008,
```

Comment:
why did you remove `var port = override && override.port || 8008` ? oh hang on, it can be removed because the override data is merged into this anyway